### PR TITLE
Docs/update

### PR DESCRIPTION
**Pull request by repjarms**
### Summary

This PR will release version `3.5.1` of the Swift SDK. It contains the following changes:

- Updates READMEs for all services (some were very outdated)
- Updates jazzy documentation generation (will generate docs for version 3.5.1
- Fixes issue with DiscoveryV2 XCode scheme that may have prevented it from building properly
- Updates jazzy static assets to latest version (resolves appscan issue with jQuery version 1.11)
- Adds Google Analytics tag to jazzy documentation

### CODE REVIEW

File: Source/SpeechToTextV1/README.md [hunk 1]
```diff
@@ -1,105 +1,96 @@
...
-
-The `SpeechToText` class is the SDK's primary interface for performing speech recognition requests. It supports the transcription of audio files, audio data, and streaming microphone data. Advanced users, however, may instead wish to use the `SpeechToTextSession` class that exposes more control over the WebSockets session.
-
-Please be sure to include both `SpeechToTextV1.framework` and `Starscream.framework` in your application. Starscream is a recursive dependency that adds support for WebSockets sessions.
-
-Beginning with iOS 10+, any application that accesses the microphone must include the `NSMicrophoneUsageDescription` key in the app's `Info.plist` file. Otherwise, the app will crash. Find more information about this [here](https://forums.developer.apple.com/thread/61521).
+* [IBM Watson Speech to Text - API Reference](https://cloud.ibm.com/apidocs/speech-to-text?code=swift)
+* [IBM Watson Speech to Text - Documentation](https://cloud.ibm.com/docs/services/speech-to-text/index.html)
+* [IBM Watson Speech to Text - Service Page](https://www.ibm.com/watson/services/speech-to-text/)
```
Comment (mediumTaj) [inline Source/SpeechToTextV1/README.md:5]: https://www.ibm.com/cloud/watson-speech-to-text

File: Source/TextToSpeechV1/README.md [hunk 2]
```diff
@@ -1,80 +1,35 @@
 # Text to Speech
 
+* [IBM Watson Text To Speech - API Reference](https://cloud.ibm.com/apidocs/text-to-speech?code=swift)
+* [IBM Watson Text To Speech - Documentation](https://cloud.ibm.com/docs/services/text-to-speech/index.html)
```
Comment (mediumTaj) [inline Source/TextToSpeechV1/README.md:4]: https://cloud.ibm.com/docs/text-to-speech/index.html

File: Source/TextToSpeechV1/README.md [hunk 3]
```diff
@@ -1,80 +1,35 @@
...
+* [IBM Watson Text To Speech - Service Page](https://www.ibm.com/watson/services/text-to-speech/)
```
Comment (mediumTaj) [inline Source/TextToSpeechV1/README.md:5]: https://www.ibm.com/cloud/watson-text-to-speech

File: Source/ToneAnalyzerV3/README.md [hunk 4]
```diff
@@ -1,5 +1,9 @@
 # Tone Analyzer
 
+* [IBM Watson Tone Analyzer - API Reference](https://cloud.ibm.com/apidocs/tone-analyzer?code=swift)
+* [IBM Watson Tone Analyzer - Documentation](https://cloud.ibm.com/docs/services/tone-analyzer/index.html)
```
Comment (mediumTaj) [inline Source/ToneAnalyzerV3/README.md:4]: https://cloud.ibm.com/docs/tone-analyzer/index.html

File: Source/ToneAnalyzerV3/README.md [hunk 5]
```diff
@@ -1,5 +1,9 @@
...
+* [IBM Watson Tone Analyzer - Service Page](https://www.ibm.com/watson/services/tone-analyzer/)
```
Comment (mediumTaj) [inline Source/ToneAnalyzerV3/README.md:5]: https://www.ibm.com/cloud/watson-tone-analyzer

File: Source/VisualRecognitionV3/README.md [hunk 6]
```diff
@@ -1,29 +1,36 @@
-# Visual Recognition
+# Visual Recognition V3
 
-The IBM Watson Visual Recognition service uses deep learning algorithms to analyze images (.jpg or .png) for scenes, objects, faces, text, and other content, and return keywords that provide information about that content. The service comes with a set of built-in classes so that you can analyze images with high accuracy right out of the box. You can also train custom classifiers to create specialized classes.
+* [IBM Watson Visual Recognition V3 - API Reference](https://cloud.ibm.com/apidocs/visual-recognition/visual-recognition-v3?code=swift)
+* [IBM Watson Visual Recognition V3 - Documentation](https://cloud.ibm.com/docs/services/visual-recognition/index.html)
```
Comment (mediumTaj) [inline Source/VisualRecognitionV3/README.md:4]: https://cloud.ibm.com/docs/visual-recognition/index.html

File: Source/VisualRecognitionV3/README.md [hunk 7]
```diff
@@ -1,29 +1,36 @@
...
+* [IBM Watson Visual Recognition V3 - Service Page](https://www.ibm.com/watson/services/visual-recognition/)
```
Comment (mediumTaj) [inline Source/VisualRecognitionV3/README.md:5]: https://www.ibm.com/cloud/watson-visual-recognition

File: Source/VisualRecognitionV4/README.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
+# Visual Recognition V4
+
+* [IBM Watson Visual Recognition V4 - API Reference](https://cloud.ibm.com/apidocs/visual-recognition/visual-recognition-v4?code=swift)
+* [IBM Watson Visual Recognition V4 - Documentation](https://cloud.ibm.com/docs/services/visual-recognition/index.html)
```
Comment (mediumTaj) [inline Source/VisualRecognitionV4/README.md:4]: https://cloud.ibm.com/docs/visual-recognition/index.html

File: Source/VisualRecognitionV4/README.md [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+* [IBM Watson Visual Recognition V4 - Service Page](https://www.ibm.com/watson/services/visual-recognition/)
```
Comment (mediumTaj) [inline Source/VisualRecognitionV4/README.md:5]: https://www.ibm.com/cloud/watson-visual-recognition

**Comment by repjarms** [comment]
@mediumTaj requested changes have been addressed. I'll merge once the new checks pass.

**Pull request merged**